mne/beamformer/tests/test_lcmv.py

```diff
@@ -139,12 +140,30 @@ def test_lcmv():
         max_stc = stc_max_power.data[idx]
         tmax = stc.times[np.argmax(max_stc)]
 
-        assert_true(0.09 < tmax < 0.11, tmax)
+        assert_true(0.04 < tmax < 0.11, tmax)
```

Comment:
how do you explain that the peak of LCMV output is not during the peak of the N100 auditory response which is between 0.08 and 0.11 seconds?